files/js/urls.ts

```diff
@@ -40,24 +40,23 @@ export const copyUrl = (url: string = window.location.href): void => {
 
 /**
  * Displays the native sharing mechanism for the device if navigator.share is available, otherwise displays a notification with the share data.
- * 
- * @param {string} url URL to share. Defaults to window.location.href.
- * @param {string} title Title of website. Defaults to window.baseTitle. 
- * @param {string} text Text to share. Defaults to window.baseDescription.
+ * @param {Object} url The `shareUrl` function requires an object with the following properties:
+ * @param {string=} url.url URL to share. Defaults to window.location.href.
+ * @param {string=} url.title Title of website. Defaults to window.baseTitle. 
+ * @param {string=} url.text Text to share. Defaults to window.baseDescription.
  */
-export const shareUrl = (url: string = window.location.href, title: string = window.baseTitle, text: string = window.baseDescription): void => {
+export const shareUrl = ({ url = window.location.href, title = window.baseTitle, text = window.baseDescription } : { url?: string, title?: string, text?: string }): void => {
 	if (navigator.share) {
 		navigator.share({ title, url }).catch(error => console.warn(error));
 	} else {
 		const textEncoded = encodeURIComponent(text + ' ' + url);
-		showNotification(
-			`<a class='button facebook' href='https://www.facebook.com/sharer/sharer.php?u=${url}' title='Share on Facebook'><i class='fas fa-share-alt'></i>&nbsp;&nbsp;Facebook</a>` +
+		showNotification({
+      text: 			`<a class='button facebook' href='https://www.facebook.com/sharer/sharer.php?u=${url}' title='Share on Facebook'><i class='fas fa-share-alt'></i>&nbsp;&nbsp;Facebook</a>` +
 			`<a class='button twitter' href='https://twitter.com/intent/tweet?text=${textEncoded}' title='Share on Twitter'><i class='fas fa-share-alt'></i>&nbsp;&nbsp;Twitter</a>` +
 			`<a class='button email' href='mailto:?subject=Check+out+${title}!&body=${textEncoded}' title='Share on Email'><i class='fas fa-share-alt'></i>&nbsp;&nbsp;Email</a>` +
 			`<a class='button copy' href='#' title='Copy' data-url='${url}'><i class='fas fa-copy'></i>&nbsp;&nbsp;Copy</button>`,
-			'',
-			'notification-share'
-		);
+      className: 'notification-share'
```

Comment:
```suggestion
      cookieName: '',
      className: 'notification-share'
```
We accidentally left out `cookieName` here :)